clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -410,73 +410,75 @@ const ConnectorParametersForm: React.FC<ConnectorParametersFormProps> = ({
                     }
                   )
                 : null}
-              {isPlusEnabled &&
-                connectionOption.supported_actions.length > 1 && (
-                  <Field
-                    id="enabled_actions"
-                    name="enabled_actions"
-                    validate={(value: string[]) => {
-                      let error;
-                      if (!value || value.length === 0) {
-                        error = "At least one request type must be selected";
+              {isPlusEnabled && (
+                <Field
+                  id="enabled_actions"
+                  name="enabled_actions"
+                  validate={(value: string[]) => {
+                    let error;
+                    if (!value || value.length === 0) {
+                      error = "At least one request type must be selected";
+                    }
+                    return error;
+                  }}
+                >
+                  {({
+                    field,
+                    form,
+                  }: {
+                    field: FieldInputProps<string>;
+                    form: any;
+                  }) => (
+                    <FormControl
+                      data-testid="enabled-actions"
+                      display="flex"
+                      isInvalid={
+                        form.touched.enabled_actions &&
+                        form.errors.enabled_actions
                       }
-                      return error;
-                    }}
-                  >
-                    {({
-                      field,
-                      form,
-                    }: {
-                      field: FieldInputProps<string>;
-                      form: any;
-                    }) => (
-                      <FormControl
-                        data-testid="enabled-actions"
-                        display="flex"
-                        isInvalid={
-                          form.touched.enabled_actions &&
-                          form.errors.enabled_actions
-                        }
-                        isRequired
+                      isRequired
+                    >
+                      {/* Known as enabled_actions throughout the front-end and back-end but it's displayed to the user as "Request types" */}
+                      {getFormLabel("enabled_actions", "Request types")}
+                      <VStack align="flex-start" w="inherit">
+                        <Box width="100%">
+                          <SelectInput
+                            options={connectionOption.supported_actions.map(
+                              (action) => ({
+                                label: _.upperFirst(action),
```

Comment:
These are still showing up in sentence case for me. If you can get access to the option element css you can use this property to make the text uppercase through css:  https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

<img width="813" alt="Screenshot 2023-11-10 at 15 02 29" src="https://github.com/ethyca/fides/assets/17103888/790524e9-b3be-4b1c-961a-64c79d5483e6">


<img width="848" alt="Screenshot 2023-11-10 at 15 02 17" src="https://github.com/ethyca/fides/assets/17103888/297bdd16-48cc-4516-a240-defcfbf63fac">